src/core.c/REPL.pm6

```diff
@@ -304,7 +304,11 @@ do {
 
         method repl-loop(*%adverbs) {
 
-            say "To exit type 'exit' or '^D'";
+            if $*DISTRO.is-win {
+                say "To exit type 'exit' or '^C'";
```

Comment:
Unix `^D` (EOF) is `^Z` on DOS-derivatives, isn't it?

Reply:
Indeed, but from a practical point of view, `^Z` requires the ENTER key to be hit afterwards on Windows, while `^C` will exit the REPL directly.
What do you propose I should move forward with? `^Z` or `^C`

Reply:
Hm... Luckily I'm not a Win user. All those inconsistencies are so confusing.

`^Z` would still be preferable because it's a request for graceful shutdown. Perhaps the message must reflect that nuance and contain `^Z Enter`?

Reply:
Done updated to ^Z